ui/marketplace/MarketplaceAppTopBar.tsx

```diff
@@ -27,7 +27,8 @@ type Props = {
 }
 
 const MarketplaceAppTopBar = ({ data, isLoading, securityReport }: Props) => {
-  const [ showContractList, setShowContractList ] = useBoolean(false);
+  const [ isContractListShown, setIsContractListShown ] = useBoolean(false);
+  const [ contractListType, setContractListType ] = React.useState(ContractListTypes.ALL);
```

Comment:
Maybe it could be combined into just one state `contractListType` with an enum or undefined value.